### PR TITLE
Article automating data engineer

### PR DESCRIPTION
**Pull request by adrianbr**

### CODE REVIEW

File: docs/website/blog/2023-06-15-automating-data-engineers.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+ **For some reason, this doesn’t work out great.** All the great minds of our generation told us we should be more like them
+
+- do more architecture
+- learn more tech
+- use this new toy
+- learn this paradigm
+- take a step back and consider your career choices.
+- write more tests
+- test the tests
```
Comment (AstrakhantsevaAA) [inline docs/website/blog/2023-06-15-automating-data-engineers.md:44]: love it =D

Comment (AstrakhantsevaAA) [reply docs/website/blog/2023-06-15-automating-data-engineers.md:44]: can we add some punctuation there? ; at least

File: docs/website/blog/2023-06-15-automating-data-engineers.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+
+>💡 “I can't do
+>   what ten people tell me to do,
+>   So I guess I'll remain the same”
+>   - Otis Redding, sitting on the dock of the bay
+
+
+
+Option 2: Take away unproductive work.
```
Comment (AstrakhantsevaAA) [inline docs/website/blog/2023-06-15-automating-data-engineers.md:57]: \### Option 2:

File: docs/website/blog/2023-06-15-automating-data-engineers.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+- **Performance optimization:** Identifying and implementing optimizations to enhance data processing and query performance, such as indexing, partitioning, and data caching.
+- **Data documentation**: Documenting data structures, data lineage, and metadata to facilitate understanding, collaboration, and data governance efforts.
+- **Data troubleshooting:** Investigating and resolving data-related issues, troubleshooting data anomalies, and providing support to resolve data-related incidents or problems.
+- **Data collaboration and sharing**: Facilitating data collaboration and sharing across teams, ensuring data accessibility, and promoting data-driven decision-making within the organization.
+- **Continuous improvement:** Staying updated with emerging technologies, industry trends, and best practices in data engineering, and actively seeking opportunities to improve data processes, quality, and efficiency.
+
+### Let’s get a back of the napkin estimation of how much time they spend on those areas.
+
+
+Here’s an approximation as offered by gpt. Of course, actual numbers depend on the maturity of your team and their unique challenges.
```
Comment (AstrakhantsevaAA) [inline docs/website/blog/2023-06-15-automating-data-engineers.md:77]: capitalize **gpt**

File: docs/website/blog/2023-06-15-automating-data-engineers.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+By looking at it this way, solutions become clear:
+
+- **Let someone else do curation.** Analysts could talk directly to producers. By removing the middle man, you improve speed and quality of the process too.
+- **Automate data structuring:** While this is not as time consuming as the collaboration, it’s the second  most time consuming process.
+- **Let analyst do exploration** of structured data at curation, not before load. This is a minor optimisation, but 10-30% is still very significant towards our goal of reducing workload by 50%.
+
+### How much of their time could be saved?
+
+Chatgpt thinks:
```
Comment (AstrakhantsevaAA) [inline docs/website/blog/2023-06-15-automating-data-engineers.md:91]: Chat GPT

File: docs/website/blog/2023-06-15-automating-data-engineers.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+1. Automation of Structuring and Maintenance: By automating the structuring and maintenance of data, data engineers **can save 30-50% or more of their time** previously spent on these tasks. This includes activities like schema evolution, data transformation, and pipeline optimization, which can be streamlined through automation.
+2. Analysts and Producers Handling Curation: Shifting the responsibility of data curation to analysts and producers **can save an additional 10-30% of the data engineer's time.** This includes tasks such as data cleaning, data validation, and data quality assurance, which can be effectively performed by individuals closer to the data and its context.
+
+It's important to note that these estimates are approximate and can vary based on the specific circumstances and skill sets within the team.
+
+# 40-80% of a data engineer’s time could be spared
+
+<aside>
+💡 **40-80% of a data engineer’s time could be spared**
```
Comment (AstrakhantsevaAA) [inline docs/website/blog/2023-06-15-automating-data-engineers.md:103]: something wrong with text bolding

**Review by rahuljo — DISMISSED** [review]

**Comment by rahuljo** [comment]
@adrianbr Love the article! Made very tiny changes. I think it's ready to merge 🚀

**Pull request merged**